include/deal.II/distributed/grid_tools.h

```diff
@@ -130,6 +131,21 @@ namespace parallel
     exchange_cell_data_to_ghosts (const MeshType &mesh,
                                   std::function<boost::optional<DataType> (const typename MeshType::active_cell_iterator &)> pack,
                                   std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> unpack);
+
+    /**
+     * Compute a vector of bounding boxes which approximates the space occupied by
+     * the locally_owned cells. The number of bounding boxes (two or more) tries
+     * to catch the general shape of the space occupied by the locally_owned_cells,
+     * describing connected components and bigger shape variations.
+     *
```

Comment:
I assume your bounding box only contains the vertices (which does not handle curved geometry). Can you leave a comment here?